src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1280,7 +1280,7 @@ def edit_zonegroup(self, realm_name: str, zonegroup_name: str, new_zonegroup_nam
     def update_period(self):
         rgw_update_period_cmd = ['period', 'update', '--commit']
         try:
-            exit_code, _, err = mgr.send_rgwadmin_command(rgw_update_period_cmd)
+            exit_code, _, err = mgr.send_rgwadmin_command(rgw_update_period_cmd, True, timeout=30)
```

Comment:
why do we set some unknown variable to `True`. Its best to change whatever we need.
```suggestion
            exit_code, _, err = mgr.send_rgwadmin_command(rgw_update_period_cmd, timeout=30)
```